cost/google/object_storage_optimization/google_object_storage_optimization.pt

```diff
@@ -226,10 +243,10 @@ policy "google_object_storage_optimization" do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): Google Object Storage Optimization."
     detail_template <<-EOS
 # Found {{ len data }} Google bucket objects which are last modified outside of the specified timeframe.
-| Bucket Name | Region | Object Name | Current Storage Class | Last Modified Date | Move Storage Class To | Tags |
-| ----------- | ------ | ----------- | --------------------- | ------------------ | --------------------- | ---- |
+| Project ID | Bucket Name | Region | Object Name | Current Storage Class | Last Modified Date | Move Storage Class To | Tags |
```

Comment:
please sort by ProjectId then region.

Reply:
I have addressed this comment please review the changes.